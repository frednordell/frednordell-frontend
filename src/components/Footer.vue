<template>
  <footer>
    <a href="https://github.com/frednordell" target="_blank">
      <img src="../assets/GitHub-Mark-Light-64px.png" alt="Github Logo" class="footer-logo">
    </a>
    <a href="https://www.linkedin.com/in/frednordell" target="_blank">
      <img src="../assets/In-White-66px-R.png" alt="LinkedIn Logo" class="footer-logo">
    </a>
  </footer>
</template>

<style scoped>
footer {
  flex-direction: row;
  -ms-flex-item-align: center;
}
.footer-logo {
  height: 2em;
  padding: 1.5em;
}
</style>
