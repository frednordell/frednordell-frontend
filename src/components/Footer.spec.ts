import "@vitest/browser/matchers.d.ts"
import { expect, test } from 'vitest'
import { render } from 'vitest-browser-vue'
import Footer from './Footer.vue'

test('should have logos in the footer', async () => {
  const { getByAltText } = render(Footer)

  await expect.element(getByAltText("GitHub Logo")).toBeInTheDocument()
  await expect.element(getByAltText("LinkedIn Logo")).toBeInTheDocument()
})

test("should match snapshot", async () => {
  const app = render(Footer);

  expect(app).toMatchSnapshot()
})