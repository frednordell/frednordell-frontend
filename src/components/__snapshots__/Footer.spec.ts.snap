// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match snapshot 1`] = `
{
  "baseElement": <body>
    
    
    
  


    <div>
      <footer
        data-v-4896eafd=""
      >
        <a
          data-v-4896eafd=""
          href="https://github.com/frednordell"
          target="_blank"
        >
          <img
            alt="Github Logo"
            class="footer-logo"
            data-v-4896eafd=""
            src="/src/assets/GitHub-Mark-Light-64px.png"
          />
        </a>
        <a
          data-v-4896eafd=""
          href="https://www.linkedin.com/in/frednordell"
          target="_blank"
        >
          <img
            alt="LinkedIn Logo"
            class="footer-logo"
            data-v-4896eafd=""
            src="/src/assets/In-White-66px-R.png"
          />
        </a>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      data-v-4896eafd=""
    >
      <a
        data-v-4896eafd=""
        href="https://github.com/frednordell"
        target="_blank"
      >
        <img
          alt="Github Logo"
          class="footer-logo"
          data-v-4896eafd=""
          src="/src/assets/GitHub-Mark-Light-64px.png"
        />
      </a>
      <a
        data-v-4896eafd=""
        href="https://www.linkedin.com/in/frednordell"
        target="_blank"
      >
        <img
          alt="LinkedIn Logo"
          class="footer-logo"
          data-v-4896eafd=""
          src="/src/assets/In-White-66px-R.png"
        />
      </a>
    </footer>
  </div>,
  "debug": [Function],
  "emitted": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholder": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
